website_sale_search_fuzzy/hooks.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, SUPERUSER_ID
+from odoo.exceptions import UserError
+
+
+def _add_trgm_index_product_tmpl_name(cr, registry):
+    with cr.savepoint():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        trgm_mod = env['trgm.index']
+
+        if trgm_mod._trgm_extension_exists() != 'installed':
+            raise UserError(_(
```

Comment:
If this logic is already included in `base_search_fuzzy`, it is superfluous here because there is no way for this module to be installed without it.